packages/frontend/src/components/ProfileHeader.tsx

```
@@ -99,132 +112,86 @@ export default function ProfileHeader({
-            <p className="text-xs text-gray-400 dark:text-gray-500 mt-1">
+            <p className="text-xs text-muted-foreground mt-1">
               Federated from {user.domain}
             </p>
           )}
         </div>
 
         {user.bio && (
           <div
-            className="note-content text-gray-700 dark:text-gray-300 mb-4"
+            className="note-content text-muted-foreground mb-4"
             dangerouslySetInnerHTML={{ __html: user.bio }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)